app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -360,7 +360,7 @@ BuilderType getTypeForBuilder(String builderName, YamlMap ci) {
 List<dynamic> _getTags(String builderName, YamlMap ci) {
   final YamlList targets = ci[kCiYamlTargetsKey] as YamlList;
   final YamlMap target = targets.firstWhere(
-    (dynamic element) => element[kCiYamlTargetBuilderKey] == builderName,
+    (dynamic element) => element[kCiYamlTargetNameKey] == builderName,
```

Comment:
This would've been caught in https://github.com/flutter/cocoon/pull/1308 if it used the proto. This isn't using the dedicated API, and is likely to be broken again. I filed https://github.com/flutter/flutter/issues/87529